assets/repertoire_comparison.Rmd

```diff
@@ -153,10 +153,21 @@ dir.create(abundance_dir)
 dir.create(vfamily_dir)
 
 # Generate one big dataframe from all patient dataframes
-
-df_list = lapply(all_files, read.csv, sep="\t")
-
-df_all <- dplyr::bind_rows(df_list)
+col_select <- c(
+  "sample_id", "subject_id", "sequence_id", "clone_id",
+  "v_call", "d_call", "j_call",
+  "locus",
+  "junction",
+  "pcr_target_locus"
+)
+i <- 0
+df_all <- dplyr::bind_rows(lapply(all_files,
+  function(x) {
+    i <<- i +1
+    message("Loading file #",i,":",x)
+    read_rearrangement(x, col_select=col_select)
+  })
+)
```

Comment:
I forgot to remove the counter (i) and message. Maybe it clutters the output if there are lots of files.

Reply:
ok will remove that now!